packages/server/src/api/rest/index.ts

```
@@ -1261,6 +1259,22 @@ class RequestHandler extends APIHandlerBase {
         return idFields.reduce((acc, curr) => ({ ...acc, [curr.name]: true }), {});
     }
 
+    private makeIdConnect(idFields: FieldInfo[], id: string | number) {
+        if (idFields.length === 1) {
+            return { [idFields[0].name]: this.coerce(idFields[0].type, id) };
+        } else {
+            return {
+                [this.makePrismaIdKey(idFields)]: idFields.reduce(
+                    (acc, curr, idx) => ({
+                        ...acc,
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)